trials_yash.ipynb

```diff
@@ -0,0 +1,641 @@
WWqVcaTGL6XsrVj9r9rxVZpXR8l9boHg6hwI\ngXe3FSHo7rGNNbucOpvapyZjEmM73orZfUqMQRkoS4XEkkwoT/KLhx+jo306ewI0gqrTZcLbZEn5\nQMyw9Mk5tgtNGUM5w6KHBnJC4QiXu/1V/nTlbSzODVMrC9S5S+iH7yYcC3BiQ/WK4sOf/mV6SkW1\nl+vI6UnM/A2SySrLJ/M8dOBsn6PuelhjJN/hlflJvHmf4Krlq0/zYyQFSVwWdEc9MNCecGzvec4m\n3PymwW/qLS6CzIXfxtM/yKLam/vJNzDY3rw3W/Nbe7cfWb9lDHyn6/2GYJbBVW/wWMN2loyehKwy\nuKHCb1qBdycShGPQHROMP59SOL/O2XO7eNc7LrCRWnjib574Sx7duIuh0SZJfpjOXdMEX36e+j6r\nEV76tiWUlpRzXbqpx8ajlgJJV4uIRKGGS6SVALdjFTHfVXqV3W5KYgwF6VAUBimsMdYzrbBpN+Vu\nfxN4jicrdyBjha6V0QWPaMQnrkA5H1GTHUoioORA2diVZUWlbGqXokgZlhIpBOtKs6R8ckIx7USs\nfXY308EN2nuKVJ6dB6AzsRthIOdmwBfXKobmnYSyG1LwEjY9QePjD1BYThDa4Jy5CkBBGfylYQoy\nZVP7KAQe1riDLOHnCQPaZsPv8KxrP+nUOeRt9G+bxoYrQzLH5TShKCSecNAIAgwS3Qf0hEZSlYrT\nscu/W/hunjhziNpLLqMvdsHYZpK4ZBt+6ofB7YDTsfMgnCwS+C7pzDCbBwIwcKk+wrunbP/BYzcO\nkihJpdSlmxRsTA94zRTtC1QkSXOgfBtr96WmM6WT7SolN8/bW5aBv8FUv62S8TcYbwkDF2yVtfqr\n3a0SCzuQQrdqw+uXHfoZ9iwWjlNkpPDrKTJ1SAqS4pLpC8l19w/hrzs8ubGfQtYQMeROcmZtitmh\nNS4WRywHV61KNAzxkCZpFkgTl5++9+t8ZvEe5o7a4+Sft1HjNeRGC8dzGDqf8PiVWR6uXsATilB7\njDstPKG5w/NBxjSz37+oHDxCvqOwivehP+TTD5zkmYW9CKHwnDr7Kw0+OPYKs16DlIB1FTHs2Lg2\nJ1I6xhAbSWg0HoIEQWhc9rhdPvziT+HEBoSg8sQVmg/O2OPWFRgHR2qGgg6u1PiZ+EFOJiTaqmnW\nzmyiSz7ORofo+D4AkoqD17JG3dQ5Gw+LhNAoalKRE3ahcYTAwdDNFqMkWwS8DFevjcETEpm1jkph\n8eeJUTZhJySdDIveMS4HHYecCLneqlF9xWXqb1fR569gAO0I8qsp3TGXtKARqcBt2fngb0Q4ixuo\nfWNENUHrQMqeQptmmiPRDkU/xhWai4tj+AqMlxFFpBqvoZA5gxEOKrA94Nq3LrNMwYnNrRNjA9wF\n/ek8UPW51biJ2WXbJvaP0EV/I5jqYFkM52Zqm8FxU01RCpC2ud/tpDiJRsYuwoAT2SaD4n85w+z8\nPl5T+3nb+62y5ecXjpFzU45VbnD64d3IJI9z337yy4ZwT0q0mUP4mr9ePs6lpVHKI+3+R2pP4oQx\nTjPCcyUqcSjKiMQ4VGRINSM1fDoSzLqKg551ETsmZlVpJDH3BcvcM/0F5sYLKCSh9nCEZo/bYCIz\nakcIVrIe66KQTDiapjasZ1xnDoaajHghqrFRL3LgVIt0vIIzv0jprKVPUrUCTlRCCsOQ37VCB9qh\nC6TaoRUGuBpUNYfTjomnKrgd+/399S7X31vmh1/+Sf7k2B9wLhkBactiHWUoy4Sy0H2oaZA11Xhi\nsKpNH28emZSCtK/VtQW49Bpsep5saFwup10OuHkmiw3C+iT64lVMmmDefpzmXp+4Kmid7CJv5Pil\n7/w8v/up77TnH8shTi1iDoxjXNh3cInD5SUut0fopD4T+SZlL+S189OIFEzJ3hfjWIVQ7VvyEOXL\nDKIMTgRumMkWbyNWHAi4M/7Abaolg7Tfr1f62qFNdpMs1xuMt4SBG7Zi7/4F+Ea/I+uZ3XnBekMY\nBrKZAlxQwum7TSLWyMDgNhMrdCcEIheQlgO8huBrF2xDhJ+zCLQvqSOM1Fo0RwqMPd1l/Csvkubv\nRz9S51fu/CL/y6kP47iK8KztijOtJZwzmzA6DAvLyPIehHQYcxsURcxhL6Ik81xPuxRFQtMIHG2B\nI4GQTDguidHMKcliWqbmdPBQ+FKRE5YAASxFU1VukSgAfZe4LBQ5IYizWvgX63fzL+//NP/m8A9T\n+/PnQQjW32aRbOU5e7zEEMiUttqij4q0i+8qOiM2l9GaLVN5Zh51w8oI4ftM/d3dbN4Y55/mf5Df\n2vdpqjLBEdDUDh3tooSiLLVt+8xurMRBY4hM0iexAEhMSiBcEhSxMUQGAiyF0+A4E03xTOhx5sY0\no02NEAI5OsragQKtPYJoVEEqGb9rmd996d0kNWt4weefxQDeRpekaHngV+ISk/kmq1GRRpJjPSog\nSwle20X1ONlcuUWymLV1CmVr3k4EbqZJfhPXWv/xQKVnZ4LtFiQPN41bcb7dxnhLGDjYlWmQuOGW\nhttnehmgr+nt5qIHeBHb3XthATLGtzeopwWeu7SG0IbWsXGcriJIpnBevEDh4HFasd1BxsY3SbQk\nVQ71Zp5kb0o8lsdLE0ZeSVgTVb4yfRQVOwhhkHF2o6SD8CGdqMKVa7jLVcT1Kf5q817uLCxw1L9I\nXYc0jcvdvseySljI+NwCoZh0DCUZcEwKjnkRkTEkbNW3e7RHkUnRaMYdmzPY0F3WtS1L7XXtc68l\nIV9s38nhwiK/eebDzHzhNeKHj6ECh+prGYXx/iJjJ5dwpCaQCW3l4wpN3knQRrCrXOfFIyXmTJnp\nxzs0HthF8S+tgXfeexfCwNjzCfIDhrbxmBAJw05ATsSsKLYh29YyWK2XsdOExpbTchmRRWIMntiq
```

Comment:
either use `file_path` in the line before, or change this `file_path` to `path`. same thing for `build_vocabulary`

Reply:
addressed.